CMakeLists.txt

```diff
@@ -2,6 +2,32 @@ cmake_minimum_required(VERSION 3.6.0)
 project(fcitx VERSION 5.0.23)
 set(FCITX_VERSION ${PROJECT_VERSION})
 
+# extra-cmake-modules
+if(DEFINED ENV{ECM_DIR})
+    set(ECM_DIR $ENV{ECM_DIR})
+else()
+	set(ECM_DIR "/usr/local/Cellar/extra-cmake-modules/5.97.0/share/ECM/cmake/")
```

Comment:
This definititely is not acceptable, we won't hardcode any version, or the path.